Prox/Prox/PlaceCarousel/PlaceCarousel.swift

```diff
@@ -87,7 +87,7 @@ extension PlaceCarousel: UICollectionViewDataSource {
         if let yelp = place.yelpProvider {
             // TODO: handle optionals in UI
             cell.yelpReview.score = Float(yelp.rating ?? -1)  // TODO: type
-            cell.yelpReview.numberOfReviewersLabel.text = "\(yelp.totalReviewCount!) Reviews"
+            cell.yelpReview.numberOfReviewersLabel.text = "\(yelp.totalReviewCount ?? -1) Reviews"
```

Comment:
Not sure I like the idea that we might tell the user there are -1 reviews. I'd rather default to 0 if we default to a number.

Alternatively we could set the text to nil in which case no review count will be displayed. Maybe @vazquez  has a preference considering his work from https://github.com/mozilla-mobile/prox/issues/38